food-truck-api/src/main/java/food/truck/api/endpoint/TruckEndpoint.java

```diff
@@ -53,16 +55,26 @@ public TruckEndpoint(TruckService truckService, RouteService routeService, Subsc
     @Autowired
     private TruckSearchService truckSearchService;
 
-    @GetMapping("/truck/nearby")
+    @PostMapping("/truck/nearby")
     public List<Truck> getNearbyTrucks(@AuthenticationPrincipal AbstractUser u) {
```

Comment:
Was there a specific reason for this to be POST? Curious since there's not `@RequestBody` or `@PathVariable`s

Reply:
Oh right. That used to need to provide a body, but Kevin streamlined the implementation and that got removed. Should we switch it back now, or wait to handle it later?

Reply:
We'll probably forget about it if we don't get it done, realistically. I don't think this should be too much of a pain to change though, since it's just changing from `api.post()` to `api.get()` and changing the annotation